server/flaskr/blueprints/before_request.py

```
@@ -18,14 +15,34 @@ def login_required():
         )
 
     if token is None:
-        return jsonify({"error": AuthError.mustBeLoggedIn}), 401
+        return None, None
 
     try:
         data = jwt.decode(token, current_app.config["SECRET_KEY"], algorithms=["HS256"])
     except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)